update.sh

```diff
@@ -9,6 +9,7 @@ supportedVersions=(
 	9.4
 	9.5
 	9.6
+	10.0
```

Comment:
This should just be 10 (with 10+, the `.0` is the minor release).  If you'd like to make the change, I'd appreciate that, otherwise I'll make it later when I get to a computer where I can merge. 👍❤️

Reply:
woops, right you are. Done.